frontend/views/components/CopyToClipboard.vue

```diff
@@ -0,0 +1,108 @@
+<template lang='pug'>
+.c-wrapper(@click='copyToClipboard')
+  input.c-invisible-input(
+    type='text'
+    ref='input'
+    :value='textToCopy'
+  )
+  slot
+  transition(name='fade')
+    .c-tooltip(v-if='ephemeral.tooltipActive') {{ L(tooltipText) }}
```

Comment:
The L function must take string literals, not variables

Reply:
Yup. I replaced this code with code using `i18n.vue`.

Reply:
In this case since you're passing it in as a property, it's being defined elsewhere. So the already-translated text is inside of `tooltipText`. So this should be:

```js
{{ tooltipText }}
```